{"abstract":[{"type":"codeVoice","code":"authError"},{"type":"text","text":" indicates an authorization failure in syncLoop or watchLoop."}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType\/authError"},"sections":[],"hierarchy":{"paths":[["doc:\/\/Yorkie\/documentation\/Yorkie","doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authError","kind":"identifier"}],"platforms":["macOS"],"languages":["swift"]}]}],"metadata":{"externalID":"s:6Yorkie12DocEventTypeO9authErroryA2CmF","symbolKind":"case","roleHeading":"Case","modules":[{"name":"Yorkie"}],"title":"DocEventType.authError","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"authError"}],"role":"symbol"},"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/yorkie\/doceventtype\/autherror"]}],"references":{"doc://Yorkie/documentation/Yorkie/DocEventType":{"abstract":[{"type":"codeVoice","code":"DocEventType"},{"type":"text","text":" is document event types"}],"url":"\/documentation\/yorkie\/doceventtype","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"DocEventType"}],"title":"DocEventType","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType","type":"topic","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"DocEventType"}]},"doc://Yorkie/documentation/Yorkie":{"type":"topic","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie","kind":"symbol","role":"collection","abstract":[],"title":"Yorkie","url":"\/documentation\/yorkie"},"doc://Yorkie/documentation/Yorkie/DocEventType/authError":{"identifier":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType\/authError","title":"DocEventType.authError","url":"\/documentation\/yorkie\/doceventtype\/autherror","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authError","kind":"identifier"}],"type":"topic","role":"symbol","abstract":[{"code":"authError","type":"codeVoice"},{"type":"text","text":" indicates an authorization failure in syncLoop or watchLoop."}],"kind":"symbol"}}}